src/oidcop/session/grant.py

```diff
@@ -27,11 +28,11 @@ class GrantMessage(ImpExp):
     }
 
     def __init__(
-        self,
-        scope: Optional[str] = "",
-        authorization_details: Optional[dict] = None,
-        claims: Optional[list] = None,
-        resources: Optional[list] = None,
+            self,
+            scope: Optional[str] = "",
+            authorization_details: Optional[dict] = None,
```

Comment:
This means that oidc-op is also aware of 
https://datatracker.ietf.org/doc/html/draft-ietf-oauth-rar

@rohe this should be exalted in the README/Official Docs
do you agree to add this feature/compliance to Documentation introduction, where we had all the specs compliances of oidc-op?

Reply:
Yes!

Reply:
here https://github.com/IdentityPython/oidc-op/pull/58